### PR TITLE
Added Label mixin for reuse across components

### PR DESCRIPTION
**Pull request by guergana**

### CODE REVIEW

**Review by jakobw — CHANGES_REQUESTED** [review]
Am aware the whole stylelint sass part isn't done yet. Just leaving these comments here for early feedback!

File: vue-components/src/styles/mixins/Label.scss [hunk 1]
```diff
@@ -0,0 +1,8 @@
+@mixin Label {
+    color: $wikit-TextInput-input-color;
```
[the remark this answers is not in the document]
Comment (guergana) [reply vue-components/src/styles/mixins/Label.scss:2]: Ok. Now I understand, yes, it makes sense!

File: package.json [hunk 2]
```diff
@@ -20,7 +20,9 @@
   },
   "devDependencies": {
     "lerna": "^3.22.1",
-    "npm-run-all": "^4.1.5"
+    "npm-run-all": "^4.1.5",
+    "stylelint": "^13.7.2",
+    "stylelint-config-sass-guidelines": "^7.1.0"
```
[the remark this answers is not in the document]
Comment (guergana) [reply package.json]: updated

File: vue-components/src/styles/mixins/Label.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (itamargiv) [reply vue-components/src/styles/mixins/Label.scss]: sonds like a good idea to me, what does @SaiSan-WMDE think?

**Comment by Ladsgroup** [comment]
A general note: Can the stylelinting be extracted to another PR? I feel they are separated stuff.

**Review by jakobw — COMMENTED** [review]
Looks good. Found one tiny code style thing to complain about, but otherwise good to go!

I agree with Amir that it's a bit hard to review because the lint changes affect the same file where the Label mixin is extracted, but looking at the commits individually helps.

File: vue-components/src/styles/mixins/Label.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (SaiSan-WMDE) [reply vue-components/src/styles/mixins/Label.scss]: Sorry for the late response! Agree to creating a new componen-level token file for the label mixin. Sounds like the best way to make it fully reusable and context-agnostic. I'd say that the later file name (tokens/properties/mixins/Label.json) makes more sense, though? It matches reality better. What do you all think?

Hate to be making @jakobw's life a nightmare lately 😅

[the remark this answers is not in the document]
Comment (guergana) [reply vue-components/src/styles/mixins/Label.scss]: Hi, it's actually me working on this ticket @SaiSan-WMDE, I added the mixin in the mixin folder

File: vue-components/stylelint.config.js [hunk 3]
```diff
@@ -11,6 +11,8 @@ module.exports = {
 		/* eslint-disable */
 		// CSS Logical Properties do not support the shorthand 'margin' and 'padding' yet
 		// https://wmde.github.io/wikit/?path=/story/documentation-decisions-and-adrs-adrs-9-rtl-support--page#consequences
-		'property-blacklist': [ '/^margin$/', '/^padding$/' ],
-	},
+		'max-nesting-depth': 3,
+		'order/properties-alphabetical-order' : null,
+		'property-disallowed-list': [ '/^margin$/', '/^padding$/' ],
+		'selector-class-pattern': null,	},
```
Comment (jakobw) [inline vue-components/stylelint.config.js:17]:
```suggestion
		'selector-class-pattern': null,
	},
```

Comment (guergana) [reply vue-components/stylelint.config.js]: It's fixed

**Review by jakobw — APPROVED** [review]
YAY :sparkles: :sparkles: :sparkles:

File: vue-components/src/styles/mixins/Label.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (SaiSan-WMDE) [reply vue-components/src/styles/mixins/Label.scss]: so that means that I'm also making your life a nightmare 🙊  Thanks, Guergana! 🙏

**Comment by guergana** [comment]
> YAY sparkles sparkles sparkles

Crying with happiness.

**Pull request merged**